rest-new/src/main/java/org/jboss/pnc/rest/endpoints/BuildConfigurationEndpointImpl.java

```diff
@@ -191,4 +203,37 @@ public BuildConfigCreationResponse createWithSCM(BuildConfigWithSCMRequest reque
     public Set<Parameter> getSupportedParameters() {
         return bcSupportedGenericParametersProvider.getSupportedGenericParameters();
     }
+
+    private Build triggerBuild(int id, OptionalInt rev, BuildParameters buildParams) {
+        try {
+            logger.debug("Endpoint /build requested for buildConfigurationId: {}, revision: {}, parameters: {}",
+                    id, rev, buildParams);
+
+            BuildOptions buildOptions = toBuildOptions(buildParams);
+            int buildId = buildTriggerer.triggerBuild(id, rev, buildOptions);
+
+            return buildProvider.getSpecific(buildId);
```

Comment:
Just a comment that this will probably break until 'buildProvider.getSpecific' implements the unified stuff between builds not in database + builds in memory

Reply:
yeah, it will.